string query_str
---
string json_str
